x-pack/legacy/plugins/siem/public/components/page/network/network_top_n_flow_table/index.tsx

```diff
@@ -122,12 +127,23 @@ const NetworkTopNFlowTableComponent = React.memo<NetworkTopNFlowTableProps>(
     let tableType: networkModel.TopNTableType;
     let headerTitle: string;
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
if anyone can help figure out this type... wrestled with it for too long on my own

Reply:
`let updateTableActivePage: NetworkTopNFlowTableDispatchProps["updateIpDetailsTableActivePage"] | NetworkTopNFlowTableDispatchProps["updateNetworkPageTableActivePage"];` maybe?

Reply:
works! Thank you

Reply:
<img width="1197" alt="Screen Shot 2019-10-11 at 8 00 36 AM" src="https://user-images.githubusercontent.com/6935300/66657550-3fbb3f00-ebfd-11e9-95df-1e72bc642373.png">
JK this fails too :(